pkg/parse/parse.go

```diff
@@ -570,25 +571,22 @@ func AuthConfig(creds string) (*types.DockerAuthConfig, error) {
 
 // GetBuildOutput is responsible for parsing custom build output argument i.e `build --output` flag.
 // Takes `buildOutput` as string and returns BuildOutputOption
-func GetBuildOutput(buildOutput string) (define.BuildOutputOption, error) {
+func GetBuildOutput(buildOutput, image string) (define.BuildOutputOption, error) {
 	if len(buildOutput) == 1 && buildOutput == "-" {
 		// Feature parity with buildkit, output tar to stdout
 		// Read more here: https://docs.docker.com/engine/reference/commandline/build/#custom-build-outputs
-		return define.BuildOutputOption{Path: "",
-			IsDir:    false,
-			IsStdout: true}, nil
-	}
-	if !strings.Contains(buildOutput, ",") {
```

Comment:
This case is now completely removed? Isn’t that a breaking change for existing users?

Reply:
It looks like a catch-call, I believe this is being handled by the switch case below, now that we expect extended attributes for types, we shouldn't be returning early here.

Reply:
Isn‘t that confirming my impression that existing syntax stops working?

Why is that acceptable? (It might well be, I’m not that familiar with Buildah. But I definitely want this to be a conscious decision by Buildah maintainers.)

Reply:
Fair enough, but now that we _are_ handling extended attributes, it'd make little sense for us to still handle this case, and besides, it's going to be counterintuitive for the user to use this. But then again, i think it's better if someone who has more context on this piece comment once.

Reply:
> that existing syntax stops working

Not entirely though, you can still go ahead and use `--output=type=image` and it'll do the right thing instead of defaulting to a preset condition as before.